Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -4730,5 +4730,32 @@
   {
     "Input": "011-12-3456 is a SSN and should not be extracted",
     "Results": []
+  },
+  {
+    "Input": "account number 12345678",
+    "Results": []
+  },
+  {
+    "Input": "account #12345678",
+    "Results": []
+  },
+  {
+    "Input": "account 12345678",
+    "Results": []
+  },
+  {
+    "Input": "card number 1234567 and phone number is +55 11 12345-6789",
```

Comment:
Please also add a more illustrative case like "card number 12345678 and phone number 12345678"

Reply:
And "My card number is 12345678"